robottelo/hosts.py

```diff
@@ -1155,17 +1098,10 @@ def configure_rhai_client(
         else:
             self.create_custom_repos(**{rhel_distro: rhel_repo})
 
-        # Install insights-client rpm
+        # Ensure insights-client rpm is installed
         if self.execute('yum install -y insights-client').status != 0:
             raise ContentHostError('Unable to install insights-client rpm')
 
-        if register_insights:
-            # Register client
-            if self.execute('insights-client --register').status != 0:
-                raise ContentHostError('Unable to register client to Insights through Satellite')
-            if self.execute('insights-client --test-connection').status != 0:
-                raise ContentHostError('Test connection failed via insights.')
```

Comment:
Why are we removing this part? Looks like we are using it, `register_insights` is set true by default.
WDYT @ColeHiggins2 ?

Reply:
The `setup_insights` param in `register` on L1079 will install and register insights-client if the user passes `register_insights=True`

Reply:
Oh I overlooked that, then we are good I guess. :+1: